prow/integ-suite-kind.sh

```diff
@@ -101,26 +101,18 @@ export CI="true"
 make init
 
 if [[ -z "${SKIP_SETUP:-}" ]]; then
-  if [[ "${TOPOLOGY}" == "SINGLE_CLUSTER" ]]; then
-    time setup_kind_cluster "${IP_FAMILY}" "${NODE_IMAGE:-}"
-  else
-    # TODO: Support IPv6 multicluster
-    time setup_kind_clusters "${TOPOLOGY}" "${NODE_IMAGE:-}"
-
-    # Set the kube configs to point to the clusters.
-    export INTEGRATION_TEST_KUBECONFIG="${CLUSTER1_KUBECONFIG},${CLUSTER2_KUBECONFIG},${CLUSTER3_KUBECONFIG}"
-    export INTEGRATION_TEST_NETWORKS="0:test-network-0,1:test-network-0,2:test-network-1"
-  fi
+  # TODO: Support IPv6 multicluster
+  time setup_kind_clusters "${TOPOLOGY}" "${NODE_IMAGE:-}" "${IP_FAMILY}"
+
+  # Set the kube configs to point to the clusters.
+  export INTEGRATION_TEST_KUBECONFIG="${CLUSTER1_KUBECONFIG},${CLUSTER2_KUBECONFIG},${CLUSTER3_KUBECONFIG}"
+  export INTEGRATION_TEST_NETWORKS="0:test-network-0,1:test-network-0,2:test-network-1"
+  export INTEGRATION_TEST_CONTROLPLANE_TOPOLOGY="0:0,1:1,2:1"
 fi
 
 if [[ -z "${SKIP_BUILD:-}" ]]; then
   time build_images
-
-  if [[ "${TOPOLOGY}" == "SINGLE_CLUSTER" ]]; then
```

Comment:
To start enabling multicluster testing in our other prow jobs we can either:
- remove the topology checks here and always perform multicluster setup
- change the prow jobs to use the MULTICLUSTER topology

Either way while this is WIP I'll leave this change in just to be able to test.